features/step_definitions/observations.rb

```diff
@@ -2,6 +2,10 @@
   expect(page.body).to include(text)
 end
 
+And(/^I do not see '(.*)'$/) do |text|
```

Comment:
This is a negation of the step `And(/^I see '(.*)'$/) do |text|` so could be combined into a single step.